src/test/java/gregtech/overclock/GT_OverclockCalculator_UnitTest.java

```diff
@@ -371,12 +377,12 @@ void testCorrectEUtWhenOverclockingUnderOneTick_Test() {
             .setDuration(56)
             .setCurrentParallel(6144)
             .calculate();
-        assertEquals((24 * 56) * Math.pow(4, 20), calculator.getConsumption());
+        assertEquals((24 * 56) * GTUtility.powInt(4, 20), calculator.getConsumption());
     }
 
     @Test
     void testCorrectEUtWhenOverclockingUnderOneTickWithHeat_Test() {
-        double heatDiscount = Math.pow(0.95, (15500 - 2000) / 900);
+        double heatDiscount = GTUtility.powInt(0.95, (15500 - 2000) / 900);
```

Comment:
Also not you, but it should be simplified so it's obvious this is an int
```suggestion
        double heatDiscount = GTUtility.powInt(0.95, 15);
```
and maybe comment where the 15 is from